browser/components/CodeEditor.js

```diff
@@ -343,6 +352,48 @@ export default class CodeEditor extends React.Component {
     })
   }
 
+  previewImage (editor, props) {
+    editor.operation(function () {
+      const storagePath = findStorage(props.storageKey).path
+      const lastline = editor.lineCount()
+      for (let i = 0; i < lastline; ++i) {
+        const lineValue = editor.getLine(i)
+        var imageMdRe = /\!\[.+\]\(\\:storage\\(.+?)\)/g
```

Comment:
This is not platform independent. I tried to develope something platform indepnendent in https://github.com/BoostIO/Boostnote/pull/1852